examples/lavalink-basic-bot/src/main.rs

```diff
@@ -113,7 +113,7 @@ async fn join(msg: Message, state: State) -> Result<(), Box<dyn Error + Send + S
             new_msg.author.id == author_id
         })
         .await?;
-    let channel_id = Id::new(msg.content.parse::<u64>()?).expect("non zero");
+    let channel_id = Id::new(msg.content.parse::<u64>()?);
```

Comment:
Spotted, might as well fix (Id now implements FromStr directly)
```suggestion
    let channel_id = msg.content.parse()?;
```